docs/root/version_history/current.rst

```diff
@@ -21,6 +21,7 @@ Minor Behavior Changes
   be now be disabled in favor of using unsigned payloads with compatible services via the new
   `use_unsigned_payload` filter option (default false).
 * cluster: added default value of 5 seconds for :ref:`connect_timeout <envoy_v3_api_field_config.cluster.v3.Cluster.connect_timeout>`.
+* grpc: grpc async client can be cached and shared accross filter instances in the same thread, this feature is turned off by default, can be turned on by setting runtime guard ``envoy.reloadable_features.enable_grpc_async_client_cache`` to true.
```

Comment:
Nit: `grpc: gRPC asyc client`, same elsewhere in this PR, generally when referring to `grpc` as a name, `gRPC` is preferrable (I also pick on `http` vs. `HTTP`).